server/routes/org_routes.py

```
@@ -115,7 +116,7 @@ def _delete_user_org_data(cursor, user_id: str, org_id: str):
         try:
             cursor.execute(f'DELETE FROM "{tbl}" WHERE user_id = %s AND org_id = %s', (user_id, org_id))
         except Exception as e:
-            logger.warning("Failed to delete %s data for user %s in org %s: %s", tbl, user_id, org_id, e)
+            logger.warning("Failed to delete %s data for user %s in org %s: %s", tbl, sanitize(user_id), sanitize(org_id), e)
```

Ruff (0.15.11)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)